lich.rbw

```diff
@@ -3236,7 +3132,6 @@ module Games
     end
 
     require_relative("./lib/spell.rb")
-    require_relative("./lib/bounty.rb")
```

Comment:
Looks like this was accidentally removed on PR to merge into existing code base. Need to add back in.

Reply:
Re-added. Thank you!